ci/requirements-py36.yml

```diff
@@ -16,6 +16,7 @@ dependencies:
     - pytest
     - pytest-cov
     - pytest-mock
+    - pytest-rerunfailures
```

Comment:
perhaps put all of the `pytest-rerunfailures` lines in the pip sections for simplicity/consistency. either way is fine though

Reply:
I was thinking of just putting py35's version in pip with the note to highlight why its there instead of conda. This stems from the idea that in the conda environment it would be ideal if everything came from conda.